packages/dd-trace/src/proxy.js

```diff
@@ -47,6 +48,7 @@ class Tracer extends BaseTracer {
 
           this._tracer = new DatadogTracer(config)
           this._instrumenter.enable(config)
+          setStartupLogPlugins(this._instrumenter._instrumented.keys())
```

Comment:
AFAICT, It only contains the ones actually loaded.

Reply:
By that point it's not yet known which plugins are loaded.

(loaded here meaning that the plugin was enabled and the corresponding module was imported since there is no lazy-loading so technically all plugins are always loaded)

Reply:
For clarity: I'm going with your definition of loaded here.

At this point when loading `acmeair`, it only showed `http`, `fs`, and `express`, which makes sense because we don't support the version of the mongodb library that it uses. That sounds to me like it's complete and correct at this point.

Reply:
What if I enable or disable a plugin after using `tracer.use`? This is a use case for a lot of users. It's also possible the plugin fails when the module is loaded. This is way too early to get this information.

Reply:
I *think* this is fixed in the latest rev.